src/libPMacc/include/algorithms/math/doubleMath/floatingPoint.tpp

```diff
@@ -60,6 +72,29 @@ struct Float2int_rd<double>
     }
 };
 
+template<>
+struct Float2int_rn<double>
+{
+    typedef int result;
```

Comment:
Offline discussion with @psychocoderHPC: The right way would be to remove result entirely and use e.g. boost::result_of or the C++11 equivalent. So any change of the name would only be a temporary fix/hack and is therefore not worth doing. Bottom line: For now `result` stays `result`